lib/route.js

```diff
@@ -970,6 +980,9 @@ function getAssociations(schema) {
     return associations;
 }
 
+route.getRestMethods = function () {
```

Comment:
Now that you've added getById, it's not _really_ restMethods, is it? Also, it's a little confusing because it looks like  it *might* return a list of HTTP methods that are allowed for a route. Perhaps call it something like getAllRestMethodNames & add a comment explaining it's usage?